Makefile

```diff
@@ -8,7 +8,7 @@ DOCKER_TAG            ?= $(USER)
 # default helm chart version must be 0.0.42 for local development (because 42 is the answer to the universe and everything)
 HELM_SEMVER           ?= 0.0.42
 # The list of helm charts needed on internal kubernetes testing environments
-CHARTS_INTEGRATION    := wire-server databases-ephemeral fake-aws nginx-ingress-controller nginx-ingress-services wire-server-metrics fluent-bit kibana sftd restund
+CHARTS_INTEGRATION    := wire-server databases-ephemeral fake-aws nginx-ingress-controller nginx-ingress-services wire-server-metrics fluent-bit kibana sftd restund coturn
```

Comment:
I would add this also to CHARTS_RELEASE on line 17 so that the chart gets published on our helm chart repository, making it easier to install it elsewhere. Right now, it would only be available from a local checkout, and wouldn't get uploaded anywhere.

Reply:
Good catch, I'd forgotten about that. I've pushed a commit which should add coturn to this list now.